ui/src/components/Homepage/index.jsx

```diff
@@ -20,6 +20,9 @@ import SignInDialog from '../SignInDialog';
     [theme.breakpoints.down('sm')]: {
       marginBottom: theme.spacing.double,
     },
+    '@media (max-width: 400px)': {
```

Comment:
Use [material-ui breakpoints](https://material-ui.com/customization/breakpoints/#breakpoints) instead of hard-coding width values. See line above for an example.